api-inference-community/docker_images/fairseq/app/pipelines/audio_to_audio.py

```diff
@@ -0,0 +1,42 @@
+import os
+from typing import Tuple, List
+
+import numpy as np
+from app.pipelines import Pipeline
+from fairseq.checkpoint_utils import load_model_ensemble_and_task_from_hf_hub
+from fairseq.models.speech_to_text.hub_interface import S2THubInterface
+
+
+class SpeechToSpeechPipeline(Pipeline):
+    def __init__(self, model_id: str):
+        model, cfg, task = load_model_ensemble_and_task_from_hf_hub(
+            model_id,
+            arg_overrides={"config_yaml": "config.yaml"},
+            cache_dir=os.getenv("HUGGINGFACE_HUB_CACHE"),
+        )
+        self.model = model[0].cpu()
+        self.model.eval()
+        cfg["task"].cpu = True
+        self.task = task
+        self.generator = task.build_generator(self.model, cfg)
```

Comment:
This lines seemed to trigger an error in previous PR, 

`elf.generator = task.build_generator(model, cfg)` fixes it but maybe changing self.model into a list of models could work too.

Reply:
Thanks for catching this! I updated accordingly and also pulled the latest main branch.